### ISSUE TITLE
Feature Idea: Disable register

### ISSUE THREAD

**Issue by loranger** [mattermost/focalboard#192]
**Summary:**
It would be convenient to being able to disable registration on self-hosted, but public focalboard.

**How important this is to me and why:**
I'd like to expose my focalboard, and still being able to invite any user, but I would like to disable /register in order to control my self-hosted focalboard usage

Importance: Medium

Use cases:
1. I install a fresh self-hosted focalboard
2. I create my own personal account
3. I edit my config and disable registration

**Comment by jespino** [comment]
@loranger actually, this works in this way, only the first account is allowed to register without invitation, after that, any user that wants to register needs to be invited. That sounds like what you expected?

**Comment by loranger** [comment]
@jespino  I tried to open my local focalboard using incognito mode and was able to fill in the registration form, but I did not try to submit it. 
If I tried, I would have seen that I'm not able to register until I provide a real valid invitation link and I would avoid you to loose time.
Sorry about this.

Let's say that I wanted to take advantage of this ticket to thank you warmly for this awesome application ! 🙏🏻
